lib/components/DaylightSavingMessage.jsx

```diff
@@ -0,0 +1,104 @@
+/*
+* == BSD2 LICENSE ==
+* Copyright (c) 2015, Tidepool Project
+*
+* This program is free software; you can redistribute it and/or modify it under
+* the terms of the associated License, which is identical to the BSD 2-Clause
+* License as published by the Open Source Initiative at opensource.org.
+*
+* This program is distributed in the hope that it will be useful, but WITHOUT
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+* FOR A PARTICULAR PURPOSE. See the License for more details.
+*
+* You should have received a copy of the License along with this program; if
+* not, you can obtain one from Tidepool Project at tidepool.org.
+* == BSD2 LICENSE ==
+*/
+
+var React = require('react');
+var sundial = require('sundial');
+
+var DaylightSavingMessage = React.createClass({
+  propTypes: {
+    onAcknowledge: React.PropTypes.func.isRequired,
+    acknowledged: React.PropTypes.number,
+    timezone: React.PropTypes.any
+  },
+  /**
+   * Set the initial state of the component
+   * 
+   * @return {Object}
+   */
+  getInitialState: function() {
+    return {
+      time: new Date().toISOString()
+    };
+  },
+
+  /**
+   * Handle the acknowledgement of the message
+   */
+  handleAcknowledge: function() {
+    this.props.onAcknowledge();
+  },
+
+  /**
+   * Determine whether the message should be displayed to the user or not
+   * 
+   * @return {boolean}
+   */
+  shouldDisplayOverlay: function() {
+    // for now show if the user has not previously acknowledged this warning
+    // in future though, we could make this show if
+    //  - the current date is within X days of DST starting/ending
+    //  - and acknowledged is false or not within X days
```

Comment:
Yeah we're going to need to adjust this for this deploy I think. We're planning to publish today, and we don't want people who update right away to acknowledge it on Saturday and then not see the warning after the switch on Sunday! Let's add `if (new Date().valueOf() >= [UTC value of switchover in selected timezone])`. Need to think about how to find that 'UTC value of switchover in selected timezone' - might not be an appropriate function in sundial.